src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const periodDiscounts = { 3: 20, 7: 50 };
+
+  const discount = getDiscount(days, periodDiscounts);
+
+  return days * pricePerDay - discount;
+}
+
+/**
+ * @param {number} days
+ * @param {Object} periodDiscounts
+ *
+ * @return {number}
+ */
+function getDiscount(days, periodDiscounts) {
```

Comment:
The function `getDiscount` is not self-explanatory and could be more descriptive. Consider renaming it to reflect that it gets the discount based on the number of rental days.